gcampus/analysis/templatetags/__init__.py

```diff
@@ -0,0 +1,14 @@
+#  Copyright (C) 2021 desklab gUG (haftungsbeschränkt)
```

Comment:
we will have to update the year of the copyright some day... 😄 But lets do that once for the entire code base when all pull requests are merged :)